brainstormforce/ultimate-addons-for-gutenberg
# Global Block Improvements

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/31a40c7d-9bf7-4856-951f-1ba10295c75e?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- Screencast added in a task comment.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Backword user
   - Drag and drop block, Add typography, Save and update the page. 
   - Switch to our branch and Typography we selected in master it should remain same after the switch.
- For new user
   -  Drag and drop the block and check the first tab from each section (General, Style, and Advanced) Should be open.
   - Set a preset and click on reset the icon. SVG color should be clear after reset.
   - Tooltip for reset, Responsive devices, and each unit in all components.
   -  Check the Cursor pointer for the font family dropdown in Typography.
   - Check Hover and Highlighted border same as elementor to indicate user which block is selected (check on responsive devices)
   - Check the Image component pointer cursor on Select, Replace and Remove by dragging any image block like the team. 
   - Check Spacing label TOP, BOTTOM, LEFT, and RIGHT misaligned is fixed or not.
   - Check debug log and console

### Checklist:
